tasks/new_e2e_tests.py

```diff
@@ -194,6 +196,46 @@ def clean(ctx, locks=True, stacks=False, output=False, skip_destroy=False):
         _clean_output()
 
 
+@task
+def cleanup_remote_stacks(ctx, pipeline_id, pulumi_backend):
```

Comment:
Is this code required as we already have cleaner doing this?

Reply:
I need this for the cleanup job I added in the CI. Since the initialization of the stack starts as soon as the pipeline is created, we are likely to never destroy the stack if for example the build of the agent fails and that we do not even try to run the e2e tests. That's why I added a cleanup job to make sure each pipeline cleans the pre-initialized clusters.
I did not want stale EKS clusters to be running for nothing waiting for the cleanup to run

Reply:
I see. I feel like the command is both not generic enough and name does not match expectations.

In current state it filters on `.pulumi/stacks/e2eci/ci-{pipeline_id}` which I think is incorrect as we specifically create a different stack name `ci-init-{job_id}`.
It also checks for `eks` in the name, which is limiting.

I believe that either we do a generic function, taking in a stack name filtering (regexp or whatever) or being named specifically to clean eks init clusters (I'd prefer first solution :))

Reply:
I updated the task to be more generic. Ideally I would have liked something that uses `pulumi stack ls --all` but that's not working at all currently because there are way too much task and the command is hanging forever. But if we fix the issue we have with the stacks, we might be able to make it cleaner!